lms/templates/provider/authorize.html

```diff
@@ -1,59 +1,66 @@
-{% extends "main_django.html" %}
+## mako
```

Comment:
It's rendered by the `django-oauth2-provider` package: https://github.com/caffeinehit/django-oauth2-provider/blob/master/provider/views.py#L104`